### PR TITLE
Updates the gitignore for VS2017

### PR DESCRIPTION
**Pull request by The-E**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -281,3 +281,6 @@ documentation/doxygen/*
 
 .idea/*
 projects/*
+/.vs
+/.vs
```
Comment (asarium) [inline .gitignore]: Why is this line duplicated?

**Comment by MageKing17** [comment]
VS2017 puts crap in the base repo folder instead of the build directory?

File: .gitignore (hunk 1 above)
Comment (The-E) [inline .gitignore]: Don't know, will correct

**Comment by The-E** [comment]
Yeah. It's part if this new functionality they introduced where vs comes with cmake built in; apparently, they need to put a couple of json files in there.

**Pull request merged**